### PR TITLE
feat(parseJson): support data v3

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Make apm compatible with data v3.

The new features of `list.json` to specify file paths and read multiple `packages.json` are not implemented in this PR.

### Progress
- [x] core.json
- [x] packages.json
- [x] list.json
- [x] Migration of dataURL
- [x] Migration of `packages.xml` in `aviutl` folder
- [x] Supports unified ID 

## Related Issue

- https://github.com/hal-shu-sato/apm/issues/387
- https://github.com/hal-shu-sato/apm-data/issues/122
  - https://github.com/hal-shu-sato/apm-data/pull/144
  - https://github.com/hal-shu-sato/apm-data/pull/159
- https://github.com/hal-shu-sato/apm-data/issues/154

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by mitosagi** [comment]
I don't know what causes CI to fail, @hal-shu-sato , do you have any idea?

**Comment by hal-shu-sato** [comment]
@mitosagi 

> I don't know what causes CI to fail, @hal-shu-sato, do you have any idea?

I think the error occurs in Ubuntu or Mac. And this PR edits `package.json`.
I think that we should use Git URL in `package.json` like this:
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#git-urls-as-dependencies

File: src/lib/parseJson.ts [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ * Add the packages to Json.
+ *
+ * @param {string} packagesListPath - A path of Json file.
+ * @param {object} packageItem - A package.
+ */
+async function addPackage(
+  packagesListPath: string,
+  packageItem: Packages['packages'][number]
+) {
+  let packages: Packages['packages'] = [];
```
Comment (yumetodo) [inline src/lib/parseJson.ts:69]:
```suggestion
  const packages: Packages['packages'] = [];
```

**Comment by mitosagi** [comment]
`@yumetodo` Thank you for the review. I have fixed that.

**Comment by hal-shu-sato** [comment]
This PR's parent is the commit before v2.2.0 was released, so could you rebase this PR?

**Comment by mitosagi** [comment]
I have rebased it.

**Review by hal-shu-sato — APPROVED** [review]
It seems to work fine.
There are many things to do for v3...

**Pull request merged**